command/harness/delegate/delegate.go

```diff
@@ -114,7 +114,7 @@ func (c *delegateCommand) run(*kingpin.ParseContext) error {
 	c.poolManager = drivers.New(ctx, instanceStore, &c.env)
 
 	_, err = harness.SetupPoolWithEnv(ctx, &c.env, c.poolManager, c.poolFile)
-	defer harness.Cleanup(c.env.Settings.ReusePool, c.poolManager, true, true) //nolint: errcheck
+	defer harness.Cleanup(c.env.Settings.ReusePool, c.poolManager, false, true) //nolint: errcheck
 	if err != nil {
```

Comment:
this will also not destroy free vms for non-ha builds

Reply:
this flag is for busyvms